README.md

```diff
@@ -24,7 +24,10 @@ See [the Configuration class](./lib/rspec_oscal_formatter/configuration.rb) for
 
 To invoke the Oscal Formatter, tests must include at least `control_id: ""` and `statement_id: ""` metadata tags. These can be given on either the `it` test or `describe` or `context` blocks.
 
-Optionally, pass `implementation_statement_uuid: 'YOUR_UUID'` to link the finding to the implementation.
+There are a couple of optional metadata pieces you can pass:
+
+* `implementation_statement_uuid: 'YOUR_UUID'` to link the finding to the implementation.
+* `rule_ids: "rule"` or `rule_ids: ["rule1", "rule2", ...]` to link the finding to Rule_Id(s) from your imported component definitions
```

Comment:
question (non-blocking): should "Rule_Id(s)" be either "`Rule_Id`(s)" or "Rule ID(s)"—or is "Rule_Id" formatted as such an OSCAL common thing?

Reply:
`Rule_Id` is the correct capitalization of how compliance-trestle names that property. I would have gone with `Rule_ID` myself. I do think that putting  it in back-ticks would help to set it off as being a specific code-level thing though